client/src/components/tools/item.jsx

```diff
@@ -178,12 +178,15 @@ export default ({ history, itemname, setLoading }) => {
           )}
         </div>
         <div className="eden_item-description">
-          {item.desc.split('\n').map((s, i) => (
-            <p key={`desc_ln_${i}`}>
-              {descriptionWithElements(s, i)}
-              <br />
-            </p>
-          ))}
+          {item.desc !== null &&
+            typeof item.desc !== 'undefined' &&
+            item.desc.split('\n').map((s, i) => (
+              <p key={`desc_ln_${i}`}>
+                {' '}
```

Comment:
Can remove this `{' '}` here.
It is an artifact created by the prettier formatter, since it can be a little eager with keeping the same amount of whitespaces before and after it has run.

Reply:
I can do that. I didn't realize prettier added that there. I'll go ahead and remove it.